src/ssh/cockpitsshtransport.c

```diff
@@ -720,13 +701,13 @@ cockpit_ssh_transport_class_init (CockpitSshTransportClass *klass)
          g_param_spec_boolean ("prompt-hostkey", NULL, NULL, FALSE,
                                G_PARAM_WRITABLE | G_PARAM_CONSTRUCT_ONLY | G_PARAM_STATIC_STRINGS));
 
-  g_object_class_install_property (object_class, PROP_CREDS,
-         g_param_spec_boxed ("creds", NULL, NULL, COCKPIT_TYPE_CREDS,
-                             G_PARAM_WRITABLE | G_PARAM_CONSTRUCT_ONLY | G_PARAM_STATIC_STRINGS));
+  g_object_class_install_property (object_class, PROP_PASSWORD,
+         g_param_spec_string ("password", NULL, NULL, NULL,
+                              G_PARAM_READWRITE | G_PARAM_CONSTRUCT_ONLY | G_PARAM_STATIC_STRINGS));
```

Comment:
We currently copy it once here (https://github.com/cockpit-project/cockpit/blob/master/src/ws/cockpitwebservice.c#L1222) in this case I'm only copying it once as well here (https://github.com/cockpit-project/cockpit/pull/6102/commits/b2508e53b2fb8accf24703f00dc2f1a8a15eff33#diff-d05217b82cb7f915e32af94329806ef5R531). I could be missing something but i think it's the same number of copies as what we do currently. Is that good enough for now.